pelicun/uq.py

```diff
@@ -1170,18 +1149,11 @@ def __init__(
             the attributes of this variable and its anchor do not have to be
             identical.
 
-        Raises
-        ------
-        ValueError
-            If there are issues with the specified distribution theta
-            parameters.
-
         """
-
         self.name = name
         self.distribution: str | None = None
         self.f_map = f_map
-        self._uni_samples: np.ndarray | None = None
+        self.uni_samples: np.ndarray | None = None
```

Comment:
Why is this plural? I think it is one sample from a uniform distribution, so it should be `uni_sample`, shouldn't it? Later, the property is indeed `uni_sample`, so this might actually be a bug.

Reply:
It doesn't look like a bug, since the property shouldn't have the same name as the hidden attribute.
Let's rename `self.uni_samples` to `self._uni_sample`.
04e64a9c